### PR TITLE
fix: change `NeynarCastCard` input types to match the API route's params

### PR DESCRIPTION
**Pull request by dylsteck**
also modifies installation instructions in `README.md`

### CODE REVIEW

**Comment by manan19** [comment]
@mrkevinoconnell to review this

File: README.md [hunk 1]
```diff
@@ -16,18 +30,24 @@ Please make sure that the following dependencies are already a part of your proj
...
   ```bash
-  npm install @neynar/react
+  yarn add react react-dom @pigment-css/react
+  ```
+
+3. Import the following CSS file in your project's root file(eg. `layout.tsx` for a Next.js app)
+  
+  ```tsx
+  import "@neynar/react/dist/style.css";
   ```
```
Comment (MrKevinOConnell) [inline README.md:51]: so much better

File: src/components/organisms/NeynarCastCard/index.tsx [hunk 2]
```diff
@@ -5,19 +5,18 @@ import { useNeynarContext } from "../../../contexts";
...
   client_id
 }: {
-  identifier: 'url' | 'hash';
-  hash?: string;
-  url?: string;
+  type: 'url' | 'hash';
+  identifier: string;
+  viewerFid?: number;
   client_id: string;
 }): Promise<any | null> {
```
Comment (MrKevinOConnell) [inline src/components/organisms/NeynarCastCard/index.tsx:17]: nice

File: src/components/organisms/NeynarCastCard/index.tsx [hunk 3]
```diff
@@ -5,19 +5,18 @@ import { useNeynarContext } from "../../../contexts";
...
   try {
-    const identifierValue = identifier === 'url' && url ? url : hash;
-    let requestUrl = `${NEYNAR_API_URL}/v2/farcaster/cast?type=${identifier}&identifier=${identifierValue}&client_id=${client_id}`;
+    let requestUrl = `${NEYNAR_API_URL}/v2/farcaster/cast?type=${type}&identifier=${identifier}${viewerFid ? `&viewer_fid=${viewerFid}` : ''}&client_id=${client_id}`;
```
Comment (MrKevinOConnell) [inline src/components/organisms/NeynarCastCard/index.tsx:19]: nice

**Review by MrKevinOConnell — APPROVED** [review]
so much better

**Pull request merged**